source/configuration_manual/sieve/plugins/enotify.rst

```diff
@@ -0,0 +1,122 @@
+.. _pigeonhole_plugin_enotify:
+
+=============================================
+Pigeonhole Sieve: Extension for Notifications
+=============================================
+
+
+The Sieve **enotify** plugin
+(`RFC5435 <http://tools.ietf.org/html/rfc5435/>`__) adds notify action to the Sieve language.
+
+Configuration
+=============
+
+The plugin is activated by adding it to the :ref:`plugin-sieve-setting-sieve_plugins` setting:
+
+::
+
+   sieve_plugins = enotify
```

Comment:
Nope

Reply:
can you hint what it should say?

Reply:
its not a plugin.. only an extension. so it cannot be added to sieve_plugins